public/index.html

```diff
@@ -49,5 +49,25 @@
   To begin the development, run `npm start` or `yarn start`.
   To create a production bundle, use `npm run build` or `yarn build`.
 -->
+<script>
+  if ('serviceWorker' in navigator) {
+    navigator.serviceWorker
+    .register('/sw.js')
+    .then((registration) => {
+      try {    
+      const sessionId = document.cookie.split(";")[0].split("=")[1]
```

Comment:
Ahhh I see. I was wondering why this was the case, but if the pronounceWords and translateWords are saved in the cookies you're right it won't work. For some reason they aren't for me, even when I toggle "See translation" and "Hear translation" on or off. I'll try and change it, so that it doesn't rely on the order, but instead looks for the actual wording "sessionID"